spec/features/create_denver_presentation_material_spec.rb

```diff
@@ -1,20 +1,106 @@
 # frozen_string_literal: true
 require 'rails_helper'
-require File.expand_path('../helpers/create_denver_work_user_context', __dir__)
-
 include Warden::Test::Helpers
 
 # NOTE: If you generated more than one work, you have to set "js: true"
 RSpec.feature 'Create a DenverPresentationMaterial', js: false do
```

Comment:
Can you check if `js: false` is set by default? I think it might be, so we could remove these arguments.